webrtc-c/canary/src/Include.h

```diff
@@ -31,6 +31,11 @@
 #define CANARY_LOG_STREAM_NAME_ENV_VAR     "CANARY_LOG_STREAM_NAME"
 #define CANARY_CERT_PATH_ENV_VAR           "CANARY_CERT_PATH"
 #define CANARY_DURATION_IN_SECONDS_ENV_VAR "CANARY_DURATION_IN_SECONDS"
+#define CANARY_BYTE_RATE_ENV_VAR           "CANARY_DATARATE_IN_BYTES_PERSECOND"
+#define CANARY_FRAME_RATE_ENV_VAR          "CANARY_FRAME_RATE"
+
+#define CANARY_METADATA_SIZE               (SIZEOF(UINT64) + SIZEOF(UINT32) + SIZEOF(UINT32))
```

Comment:
Instead of this, we can define a structure in C and simply use SIZEOF(CanaryMetadata);

Reply:
As per your comment, and I think the same, it is better to deal with the actual bits rather than with a structure at this point, owing to endianness across platforms.